core/src/main/res/values/strings.xml

```diff
@@ -850,4 +850,9 @@
     <string name="on_demand_config_setting_changed">Setting updated successfully.</string>
     <string name="on_demand_config_stream_text">Looks like you stream a lot. Do you want episode lists to show stream buttons?</string>
     <string name="on_demand_config_download_text">Looks like you download a lot. Do you want episode lists to show download buttons?</string>
+
+    <string name="shortcut_unsupported_device">Device does not support creating shortcuts.</string>
+    <string name="shortcut_subscription_label">Subscription</string>
```

Comment:
I would maybe call this something like "Subscription shortcut" or something. Just "subscription" could be hard to understand because many phones list shortcuts and widgets in the same list